src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,90 @@
+    ```
+
+1. Send the chain’s gateway and prover addresses to the Interop Labs team by filling out the [Google form](TODO: ADD LINK). The team will register the chain and gateway with the router and authorize the prover.
+
+## Begin routing messages
+
+Once your chain is registered with the protocol, it will need verifiers to begin routing messages. See [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for instructions on onboarding a verifier.
```

Comment:
```suggestion
Once your chain is registered with the protocol, it will need verifiers to begin routing messages. See [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for instructions on onboarding a verifier.

Once you have onboarded your verifiers, run the following to actually register your verifiers with the deployed prover:

axelard tx wasm execute axelar1s4jwm729da3u9hr9vej8mw65uuzzwvwrdnp6l0rk73m0xef2grzq3m0mrw '"update_worker_set"' --from wallet --chain-id devnet-amplifier --gas auto --gas-adjustment 1.5 --gas-prices 0.007uamplifier --node http://devnet-amplifier.axelar.dev:26657

If at any time later, you would like to change the verifier set (register new verifiers and/or remove existing verifiers), you must perform a worker set update to begin using the new set. See the in repo documentation for more details about this process: https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/multisig_prover.md#update-and-confirm-workerset-graph
```

Reply:
forgot a step, please add @ffe9f8